witness/src/Dscp/Witness/Launcher/Mode.hs

```diff
@@ -74,12 +79,13 @@ type WitnessWorkMode ctx m =
 
 -- | Context is resources plus some runtime variables.
 data WitnessContext = WitnessContext
-    { _wcParams     :: !WitnessParams     -- ^ Parameters witness was started with.
-    , _wcResources  :: !WitnessResources  -- ^ Resources, allocated from params.
-    , _wcMempool    :: !MempoolVar
-    , _wcSDActions  :: !SDActions
-    , _wcRelayState :: !RelayState
-    , _wcSDLock     :: !SDLock
+    { _wcParams          :: !WitnessParams     -- ^ Parameters witness was started with.
+    , _wcResources       :: !WitnessResources  -- ^ Resources, allocated from params.
+    , _wcMempool         :: !MempoolVar
+    , _wcSDActions       :: !SDActions
+    , _wcRelayState      :: !RelayState
+    , _wcSDParamsBuilder :: !SD.OSParamsBuilder
```

Comment:
Rename `SDActions` into `SDVars` and put this `OSParamsBuilder` inside.

Reply:
@astiunov please fix